tests/cli/test_deploy.py

```diff
@@ -4807,36 +4788,19 @@ def test_save_user_inputs_with_interval_schedule(self):
     def test_save_user_inputs_with_cron_schedule(self):
         invoke_and_assert(
             command="deploy flows/hello.py:my_flow",
-            user_input=(
-                # Accept default deployment name
-                readchar.key.ENTER
-                +
-                # accept schedule
-                readchar.key.ENTER
-                +
-                # select cron schedule
-                readchar.key.DOWN
-                + readchar.key.ENTER
-                # enter cron schedule
-                + "* * * * *"
-                + readchar.key.ENTER
-                # accept default timezone
-                + readchar.key.ENTER
-                # accept schedule being active
-                + readchar.key.ENTER
-                # decline adding another schedule
-                + readchar.key.ENTER
-                # accept create work pool
-                + readchar.key.ENTER
-                # choose process work pool
-                + readchar.key.ENTER
-                # enter work pool name
-                + "inflatable"
-                + readchar.key.ENTER
-                # accept save user inputs
-                + "y"
-                + readchar.key.ENTER
-            ),
+            prompts_and_responses=[
+                ("? Deployment name (default)", ""),
+                ("Would you like to configure schedules for this deployment?", ""),
+                ("What type of schedule would you like to use?", "↓", "Cron"),
```

Comment:
This is an example of using the down arrow (`↓`) to select an option in the table. You can also use `readchar.key.DOWN` here if you prefer.